CHANGELOG.md

```diff
@@ -1,3 +1,74 @@
+# v1.0.6 (2021-02-26)
+
+## OS Changes
```

Comment:
nit: for consistency, let's have a blank line under each heading, like past releases.